ignite/templates/module/create/files/base/x/{{moduleName}}/module/module.go.plush

```diff
@@ -94,9 +96,9 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *r
 // GetTxCmd returns the root Tx command for the module.
 // These commands enrich the AutoCLI tx commands.
 // When creating non AutoCLI commands, add the following:
-// func (a AppModuleBasic) GetTxCmd() *cobra.Command {
-//    return cli.GetTxCmd()
-// }
+func (a AppModuleBasic) GetTxCmd() *cobra.Command {
+    <%= if (isIBC) { %> return cli.GetTxCmd() <% } else { %>return nil<% } %>
```

Comment:
Can we wrap the whole GetTxCmd in there? This way non IBC module won't have anything commented either.